lib/KeyboardAwareMixin.js

```diff
@@ -89,8 +92,14 @@ const KeyboardAwareMixin = {
 
   componentDidMount: function () {
     // Keyboard events
-    this.keyboardWillShowEvent = Keyboard.addListener('keyboardWillShow', this.updateKeyboardSpace)
-    this.keyboardWillHideEvent = Keyboard.addListener('keyboardWillHide', this.resetKeyboardSpace)
+    if (Platform.OS == 'ios') {
+      this.keyboardWillShowEvent = Keyboard.addListener('keyboardWillShow', this.updateKeyboardSpace)
+      this.keyboardWillHideEvent = Keyboard.addListener('keyboardWillHide', this.resetKeyboardSpace)
+    }else {
+      this.keyboardWillShowEvent = Keyboard.addListener('keyboardDidShow', this.updateKeyboardSpace)
```

Comment:
@alvaromb & @grsabreu  You can directly replace keyboardWillShow & keyboardWillHide with keyboardDidShow and keyboardDidHide . You don't need to check the platform  **keyboardDidShow and keyboardDidHide** for both IOS and android